apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -202,6 +208,31 @@ export class SubscribersController {
     );
   }
 
+  @Patch('/:subscriberId/online-status')
```

Comment:
As is a specific status, I find it ok to do that and you endpoint looks great. 👍🏻 

As a suggestion, what you might consider is if you would prefer a more _action_ oriented endpoints such:
```
POST /:subscriberId/online-status/connected
POST /:subscriberId/online-status/disconnected
```
Overall it would only be 2 extra methods in the controller and both could share the use case as you can define the flag value in the method controller, and skip the body.

Reply:
I'll stick with the current one, thank you for the suggestion will keep it in mind :)